dragula.js

```diff
@@ -404,11 +404,44 @@ function dragula (initialContainers, options) {
       dropTarget.insertBefore(item, reference);
       drake.emit('shadow', item, dropTarget, _source);
     }
+
+    var h = window.innerHeight;
+    document.addEventListener('mousemove', function(e) {     
+        var scrollTop = (window.pageYOffset || document.scrollTop)  - (document.clientTop || 0);
```

Comment:
It's NaN in my case running on latest Chrome. I think the problem is that `(0 || undefined)` will be `undefined`. Btw. it would be nice to check the coordinates of the parent element instead of the document because often the parent element has a scrollbar not the whole page.